website/content/docs/connect/failover/sameness.mdx

```diff
@@ -0,0 +1,206 @@
+---
+layout: docs
+page_title: Failover with sameness groups
+description: You can configure sameness groups so that when a service instance fails, traffic automatically routes to an identical service instance. Learn how to use sameness groups to create a failover strategy for deployments with multiple datacenters and cluster peering connections.
+---
+
+# Failover with sameness groups
+
+This page describes how to use sameness groups to automatically redirect service traffic to healthy instances in failover scenarios. Sameness groups are a user-defined set of Consul admin partitions with identical registered services. These admin paritions typically belong to Consul datacenters in different cloud regions, which enables sameness groups to participate in several service failover configuration strategies.
+
+To create a sameness group and configure each Consul datacenter to allow traffic from other members of the group, refer to [create sameness groups](/consul/docs/connect/cluster-peering/usage/create-sameness-groups).
+
+## Failover strategies
+
+You can edit a sameness group configuration entry so that all services failover to healthy instances on other members of a sameness group by default. You can also reference the sameness group in other configuration entries to enact other failover strategies for your network. 
+
+You can establish a failover strategy by configuring sameness group behavior in the following locations:
+
+- Sameness group configuration entry
+- Service resolver configuration entry
+- Prepared queries
+
+### Failover with a sameness group configuration entry
+
+To define failover behavior using a sameness group configuration entry, set `DefaultForFailover=true` and then apply the updated configuration to all clusters that are members of the group.
+
+When `DefaultForFailover=true`, Consul attempts to redirect traffic to healthy instances according to the order of the group `Members` in the configuration entry. When you set `IncludeLocal=true` in the sameness group configuration entry, Consul attempts to redirect traffic to another instance in the same admin partition before redirecting to other group members.
+
+In the following example configuration entry, datacenter `dc1` has two partitions, `partition-1` and `partition-2`. A second datacenter, `dc2`, has a single partition named `partition-1`. All three partitions have identically configured services and established cluster peering connections. The configuration entry defines a sameness group, `example-sg` in `dc1`. When redirecting traffic during a failover scenario, Consul will attempt to find a healthy instance in a specific order: `dc1-partition-1`, then `dc-1-partition-2`, then `dc2-partition-1`.
+
+<Tabs>
+
+<Tab heading="HCL" group="hcl">
+
+```hcl
+Kind                     = "sameness-group"  
+Name                     = "example-sg"
+Partition                = "partition-1" 
+DefaultForFailover       = true
+IncludeLocal             = true
```

Comment:
`IncludeLocal=true` here will cause an error because `partition-1` is already defined in the member list.